### PR TITLE
dietpi-software command line install option described

### PR DESCRIPTION
**Pull request by StephanStS**

### CODE REVIEW

File: docs/software/gaming.md [hunk 1]
```diff
@@ -168,7 +168,7 @@ This installation is possible due to a collaboration with Dimitris Panokostas (A
 
     - Kickstarts (Amiga BIOS/boot system)  
       Amiga Kickstart ROM images are required to run the system(s) you want to emulate. These can not be bundled due to copyright restrictions.  
-      If you own the Amiga Forever product, you can legally download and use Kickstarts you are eligible for, from: <https://www.amigaroms.net/>.  
+      If you own the Amiga Forever product, you can legally download and use Kickstarts you are eligible for, from: <https://www.amigaroms.com/>.
```
Comment (MichaIng) [inline docs/software/gaming.md:171]: Just fixed it in the forum as well 👍.

File: docs/software/gaming.md [hunk 2]
```diff
@@ -242,49 +242,53 @@ When ready, select Start to launch the emulator. Have fun!
 
 ### FAQ
 
-- How can I transfer Kickstarts & Floppy Images to the device?  
-  Make sure you have one of DietPi's File Servers installed.
+**How can I transfer Kickstarts & Floppy Images to the device?**
```
Comment (MichaIng) [inline docs/software/gaming.md:245]: This needs to be a header to satisfy Markdownlint, and `####` anyway has the same visual result, so should be fine. Same for below cases.

File: docs/software/gaming.md [hunk 3]
```diff
@@ -242,49 +242,53 @@ When ready, select Start to launch the emulator. Have fun!
...
 ### FAQ
 
-- How can I transfer Kickstarts & Floppy Images to the device?  
-  Make sure you have one of DietPi's File Servers installed.
+**How can I transfer Kickstarts & Floppy Images to the device?**  
+Make sure you have one of DietPi's File Servers installed.
 
-    - Floppy Disk Image (.adf) directory as chosen before, e.g. `/amiberry/floppy_images`
-    - Kickstarts (.rom) directory = `/amiberry/kickstarts`
+- Floppy Disk Image (.adf) directory as chosen before, e.g. `/amiberry/floppy_images`
```
Comment (MichaIng) [inline docs/software/gaming.md:248]: Trailing white space.

File: docs/software/gaming.md (hunk 2 above)
Comment (StephanStS) [reply docs/software/gaming.md:245]: Nearly the same: Same font size but larger distance between the lines.  
Changed all the FAQ entries.

File: docs/software/gaming.md (hunk 3 above)
Comment (StephanStS) [reply docs/software/gaming.md:248]: Yep. Changed.

**Pull request merged**